packages/core/src/utilities/background-color/_background-color.docs.scss

```diff
@@ -3,7 +3,7 @@ Background color
 
 Use the background color utility to change the default background color of an element.
 
-**Format**: `ds-u-fill--[COLOR NAME]-[SHADE]`
+**Format**: `.ds-u-fill--[COLOR NAME]-[SHADE]`
```

Comment:
Adding `.` in as that is the preferred format

Reply:
Agreed, we should file a ticket to make this change sitewide

Reply:
I'll file the ticket now 👍